.changeset/green-maps-end.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/button": minor
+---
+
+Adding the ability to disable label wrap in button component
```

Comment:
```suggestion
Adds the ability to disable text wrapping within the button component's label by applying the class `.spectrum-Button--noWrap` to `.spectrum-Button`.
```
Changing the tense and adding some more detail here, since this will go in the change log and be referenced by SWC and others.